src/scalib_ext/ranklib/src/histogram.rs

```diff
@@ -7,11 +7,17 @@ mod bnp {
     include!(concat!(env!("OUT_DIR"), "/binding_bnp.rs"));
 }
 
+const CONV_LIMIT: f64 = 34359738368.0 as f64; //define a certain limit
```

Comment:
Please explain the choice of this value in more detail

Reply:
Can you explain why it happens at 2^70 and why your solution works in more detail? You need to also edit the comment to explain that constant in more detail.